presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -293,7 +294,10 @@ public PrestoSparkQueryRunner(String defaultCatalog, Map<String, String> additio
         pluginManager.installPlugin(new HivePlugin("hive", Optional.of(metastore)));
 
         connectorManager.createConnection("hive", "hive", ImmutableMap.of(
-                "hive.experimental-optimized-partition-update-serialization-enabled", "true"));
+                "hive.experimental-optimized-partition-update-serialization-enabled", "true",
+                "hive.allow-drop-table", "true", "hive.allow-rename-table", "true",
```

Comment:
I am trying to understand why this was necessary to test DDL when we don't use this approach to configure security in any other tests. It seems like HiveQueryRunner generally uses "sql-standard".

That sent me down the rabbit hole of trying to understand what security OSS PoS uses and it seems like it would get the default "legacy".

Reply:
nit: one property per line

Reply:
I am getting access denied exception if I don't set these properties 
`com.facebook.presto.spi.security.AccessDeniedException: Access Denied: Cannot drop table XXXX
	at com.facebook.presto.spi.security.AccessDeniedException.denyDropTable(AccessDeniedException.java:116)
	at com.facebook.presto.spi.security.AccessDeniedException.denyDropTable(AccessDeniedException.java:111)
	at com.facebook.presto.hive.security.LegacyAccessControl.checkCanDropTable(LegacyAccessControl.java:100)
	at com.facebook.presto.hive.security.SystemTableAwareAccessControl.checkCanDropTable(SystemTableAwareAccessControl.java:88)
	at com.facebook.presto.security.AccessControlManager.lambda$checkCanDropTable$20(AccessControlManager.java:304)
	at com.facebook.presto.security.AccessControlManager.authorizationCheck(AccessControlManager.java:738)
	at com.facebook.presto.security.AccessControlManager.checkCanDropTable(AccessControlManager.java:304)
	at com.facebook.presto.execution.DropTableTask.execute(DropTableTask.java:67)
`